wallet/reference/multichain-api.md

```diff
@@ -383,4 +383,18 @@ according to [CAIP-217](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/c
     }
   }
 }
-```
\ No newline at end of file
+```
+
+## Error codes
+
+The Multichain API can return the following JSON-RPC error codes:
+
+| Code | Description | Related methods |
+| ---- | ----------- | --------------- |
+| 5000 | Unknown error with request | `wallet_revokeSession`, `wallet_createSession` | 
+| 5100 | Requested networks are not supported | `wallet_createSession` |
+| 5101 | Requested methods are not supported | `wallet_createSession` |
+| 5102 | Requested notifications are not supported | `wallet_createSession` |
+| 5300 | Invalid `scopedProperties` requested | `wallet_createSession`|
```

Comment:
we aren't technically firing this yet. thoughts, @adonesky1 ?

Reply:
Lol do we fire any of these errors? 5100, 5101, 5102, 5300? I'm pretty sure we've decided all of these are way too restrictive

Reply:
I also don't see 5300 or 5301 thrown anywhere in our code

Reply:
We should remove these from [the OpenRPC doc](https://github.com/MetaMask/api-specs/blob/f5e94e75b3434c284501810e8043484b4e697c03/multichain/openrpc.yaml#L567):

Reply:
The only one of these errors I can see us actually throwing all the way through to the dApp [is 5302](https://github.com/MetaMask/core/blob/31b79731139fa55960ef4e56546b33bbee966b8a/packages/multichain-api-middleware/src/handlers/wallet-createSession.ts#L99)

Reply:
Sorry actually we [also throw 5100](https://github.com/MetaMask/core/blob/31b79731139fa55960ef4e56546b33bbee966b8a/packages/multichain-api-middleware/src/handlers/wallet-createSession.ts#L232)

so its just 5100 and 5302